README.md

```diff
@@ -26,3 +26,23 @@ To install [pinocchio](https://github.com/stack-of-tasks/pinocchio) follow the i
     cd _build-RELEASE
     cmake .. -DCMAKE_BUILD_TYPE=RELEASE -DCMAKE_INSTALL_PREFIX=$DEVEL/openrobots
     make install
+
+## TSID for Python 
+To operate it in python environment, we offer python binding module using boost::python and eigenpy
```

Comment:
Here Boost.Python and EigenPy for the formating.

Reply:
Fixed it!.